  Fixed bug: resolution task wasn't recreated  The old resolved calls were used  